rtos/rtx5/TARGET_CORTEX_M/rtx_lib.c

```diff
@@ -595,7 +595,6 @@ void *__user_perthread_libspace (void) {
 typedef void *mutex;
 
 // Initialize mutex
-__USED
```

Comment:
Why was `__USED` removed for _mutex_initialize only and not the other functions like _mutex_free?